pkg/types/conf_test.go

```diff
@@ -928,7 +928,13 @@ var _ = Describe("config operations", func() {
 
 		netconf, err := LoadDelegateNetConf([]byte(conf), ns, "", "")
 		Expect(err).NotTo(HaveOccurred())
+
+		err = CheckGatewayConfig([]*DelegateNetConf{netconf})
+		Expect(err).NotTo(HaveOccurred())
```

Comment:
nit: You could also inline these two - I think it makes more sense to use positive logic, unless it is impossible.
```golang
Expect(CheckGatewayConfig([]*DelegateNetConf{netconf})).To(Succeed())
```